simptome.json

```diff
@@ -0,0 +1,42 @@
+{"febra": 0,
```

Comment:
Stabilisem ca vom tine valoarea 0 pentru simptome necunoscute si in caz ca nu are niciun simptom. Poti sa faci sa inceapa indexarea de la 1, nu de la 0?